modules/core/src/main/java/org/apache/ignite/cache/CacheMetrics.java

```diff
@@ -720,4 +720,18 @@
      * @return Key collisions and appropriate queue size string representation.
      */
     @NotNull public String getTxKeyCollisions();
+
+    /**
+     * Return {@code true} if index rebuild is in progress.
+     *
+     * @return {@code true} if index rebuild is in progress.
+     */
+    public boolean isIndexRebuildInProgress();
+
+    /**
+     * Return number of keys processed during index rebuilding.
```

Comment:
Please explain as clear as possible how metric result should be interpreted. We need to link method that retrieves local cache size here.
Also, wouldn't it be better to name this method getIndexRebuildKey**s**Processed? We process multiple keys, not a single one.